control-plane-operator/controllers/hostedcontrolplane/ignitionserver/ignitionserver.go

```diff
@@ -843,3 +879,32 @@ cp /tmp/manifests/99_feature-gate.yaml %[1]s/99_feature-gate.yaml
 `
 	return fmt.Sprintf(script, workDir, featureGateYAML)
 }
+
+func lookupDisconnectedRegistry(ctx context.Context, strOcpOverrides string) string {
```

Comment:
nit: this could be removed since it's not used.

Reply:
That's where it's passed in, but it's not used within the function at all unless I missed it.